### PR TITLE
Readme proofreading/formatting

### PR DESCRIPTION
**Pull request by evan-masseau**
Did a pass over the readme again since I was a bit distracted reviewing the earlier PR

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -136,20 +137,21 @@ pod install
...
+Note: It is safe to re-initialize, e.g. if your app needs to switch between more than one Klaviyo account.
 
 ### React Native Initialization
 
 Below is an example of how to initialize the SDK from your React Native code:
 
 ```typescript
 import { Klaviyo } from 'klaviyo-react-native-sdk';
-Klaviyo.initialize('YOUR_KLAVIYO_PUBLIC_API_KEY');
+Klaviyo.initialize('YOUR_PUBLIC_KLAVIYO_API_KEY');
```
Comment (evan-masseau) [inline README.md:154]: the smallest nitpick, but this is the value we used in the code.

File: README.md [hunk 2]
```diff
@@ -339,49 +339,50 @@ Follow the platform-specific instructions below:
...
-   - [Android](https://github.com/klaviyo/klaviyo-android-sdk#collecting-push-tokens)
-   - [iOS](https://github.com/klaviyo/klaviyo-swift-sdk?tab=readme-ov-file#request-push-notification-permission)
-
-   If you requested permission using native code then continue using Klaviyo's native platform SDKs `setToken` method to inform the SDK of permission change.
+Requesting user permission to display notifications can be managed from the React Native code, or from platform-specific
+native code. Note that either of these approaches is sufficient to inform the Klaviyo SDK of the permission change.
 
-2. Leveraging a third party library that provides cross-platform permissions APIs like firebase [`react-native-firebase/messaging`](https://www.npmjs.com/package/@react-native-firebase/messaging). If you opt for a
-   cross-platform permission solution, you can now call the Klaviyo's react native SDK's `setToken` method to refresh the token's enablement status.
+1. **React Native Notification Permission**:
```
Comment (evan-masseau) [inline README.md:345]: I figured for consistency we should put react native section first

File: README.md [hunk 3]
```diff
@@ -444,7 +445,7 @@ Linking.getInitialURL().then((url) => {
 
 ## Troubleshooting
 
-Use the [troubleshooting guide](Troubeshooting.md) to resolve common issues with the Klaviyo React Native SDK.
+Use the [troubleshooting guide](Troubleshooting.md) to resolve common issues with the Klaviyo React Native SDK.
```
Comment (ajaysubra) [inline README.md:448]: 🤦🏽

Comment (ajaysubra) [inline README.md:448]: 🤦🏽

**Review by ajaysubra — APPROVED** [review]
Thanks!

**Pull request merged**